frontend/src/components/features/settings/brand-button.tsx

```diff
@@ -2,6 +2,7 @@ import { cn } from "#/utils/utils";
 
```

Comment:
does this branch button component also exist for api settings? i think i remember seeing this in another PR